requirements/base.in

```diff
@@ -2,7 +2,7 @@
 
 -c constraints.txt
 
-Django>=1.11,<2.0          # Web application framework
+Django>=1.11,<2.3          # Web application framework
```

Comment:
I think instead of adding the constraint in the file below, we just want to split this one into two parts:

* `Django>=1.11`
* `Django<2.0 ; python_version < "3"`

The upper constraint was just because of the incompatibility of Django 2.0+ with Python 2, we don't normally want to impose such limits.  Also, you'll need to re-run `make upgrade` after changing this file.

I was originally thinking of moving the upper limit to the constraints file, but I don't think that will work with how we currently have `setup.py` set up to parse the dependencies.

Reply:
@jmbowman I have run the make upgrade using python2.7 env.